app/assets/javascripts/discourse/app/components/d-navigation.js

```diff
@@ -95,5 +107,24 @@ export default Component.extend(FilterModeMixin, {
         this.createTopic();
       }
     },
+
+    toggleTagInfo() {
+      return this.toggleProperty("showInfo");
+    },
+
+    changeTagNotificationLevel(notificationLevel) {
```

Comment:
Are you moving this action from somewhere else? Does it need to be removed from the other place?

If not, we can pass in the action as a parameter so that we're not repeating ourselves.